Tmain/omit-long-patterns-etags.d/stdout-expected.txt

```diff
@@ -0,0 +1,5 @@
+
+input.sh,349
+                                                                                                func96(func961,0
```

Comment:
@dgutov I though the goal was to truncate after N characters, and add the name in a separate field, so what @masatake did originally seemed correct.

Of course with the spaces example it won't be a very useful search line, but that's pretty artificial IMO.